.github/actions/validate.yaml

```diff
@@ -0,0 +1,33 @@
+name: 'Validate'
+description: 'Validate kubernetes manifests using kustomize build'
+inputs:
```

Comment:
You need to change to: 
```
on:
  workflow_call:
    inputs:
```
Look at: https://github.com/XenitAB/azure-devops-templates/blob/main/.github/workflows/push-image-acr.yaml

Reply:
Well, the intention was not create a workflow but a reusable action

Reply:
See: https://github.com/XenitAB/gitops-promotion/blob/main/action.yaml

Reply:
But I will probably have to move this one to it's own repo